pkg/nvmdev/nvmdev.go

```diff
@@ -322,7 +322,7 @@ func (m *Device) Delete() error {
 
 // GetPhysicalFunction gets the physical PCI device backing a 'parent' device.
 func (p *ParentDevice) GetPhysicalFunction() (*nvpci.NvidiaPCIDevice, error) {
-	if !p.IsVF {
+	if !p.IsVF() {
```

Comment:
Can we use `isPF()` instead?

Reply:
We can simplify even more to:
```go
func (p *ParentDevice) GetPhysicalFunction() *nvpci.NvidiaPCIDevice {
	if p.IsVF() {
		return p.SriovInfo.PhysicalFunction
	}
	return p.NvidiaPCIDevice
}

```
I guess not all ParentDevices has to be SRIOV devices.